src/main/java/org/folio/services/batch/BatchOperationContextFactory.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.services.batch;
+
+import static io.vertx.core.Future.succeededFuture;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.toList;
+
+import io.vertx.core.Future;
+import java.util.List;
+import java.util.function.Function;
+import org.folio.persist.AbstractRepository;
+
+public final class BatchOperationContextFactory {
+  private BatchOperationContextFactory() {}
+
+  public static <T> Future<BatchOperationContext<T>> buildBatchOperationContext(
```

Comment:
Consider to update `InstanceService` line 76 to use this to reduce code repeat.

Reply:
Thank you! Changed now.